.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+The membership portal is an open-source project, for members by members, and we welcome any contributions! This project is maintained by ACM UCSD's development team.
+
+### Getting Started
+Issues in our projects are kind of like conversations; when we find a problem or propose an idea, we open an issue and discuss in the comments.  **You can look through our open issues [here](https://github.com/acmucsd/membership-portal/issues)**; if you see something you're interested in working on, feel free to leave a comment with any questions you might have, and one of our maintainers will follow up with you. **You can also message `@Development` on the ACM UCSD Discord** and ask about our current priorities.
+
+**If you're here on behalf of another team looking to use this API, e.g. a side project or some other app, message `@Development` on the main Discord server and schedule a meeting with our backend team to discuss API use cases** so we're aware of and can plan for any incoming traffic from your app. And, when considering the entire tech architecture of our chapter, the features you're looking to build on top of the API in your app might better fit within the application boundaries of the portal itself; that's been the case for many API clients historically. Future teams can make use of that same functionality, and we can avoid disconnected apps and data siloes, as our membership data is deeply useful for building a higher-quality organization.
+
+**Please make sure to discuss your approach with a maintainer before starting any work** so that we can catch any major issues early. **Once you're ready to start coding, check out the README for instructions on getting the API running locally.** If any of our documentation is unclear or outdated, or you think our codebase could be doing something better, please suggest changes to make things easier for the next contributor!
+
+Once your functionality is implemented, open up a PR and a maintainer will review it as soon as possible. Anecdotally, most of our bigger PRs go through several rounds of review before merging, so if you're implementing a feature for an API client, be sure to start early, start often to meet your team's deadlines. If you're stuck or confused at any point, please don't hesitate to ask one of our maintainers for help.
+
+### Project Structure
+There're a lot of moving parts in this codebase and making simple changes can be daunting as a first-time contributor, so this section breaks down all the areas of the codebase you might need to edit to make your changes.
+
+#### API Specification
+You can find our API specification [here](https://documenter.getpostman.com/view/12949536/TVRedVwQ). It'll tell you about all the endpoints our API exposes and what each does. All the types in our specification, e.g. requests, responses, user profiles, merch orders, are defined in the `types` folder, and if you're adding a route or modifying an existing one, you'll first need to edit these types.
```

Comment:
I think you should bold the sentence "All the types in our specification... defined in the `types` folder", since I think it's super important to highlight that `types` should be the starting point for any large feature update to a newcomer

Reply:
The project structure section's in chronological order for changes to make. I mostly bolded sentences in the Getting Started section since people might skim over that, but if you're a newcomer you'll probably read the project structure section in its entirety.